src/components/Proposal/View/ProposalDescriptionItem.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+import Markdown from 'decentraland-gatsby/dist/components/Text/Markdown'
+
+interface Props {
+  title: string
+  body: string | string[]
+}
+
+function ProposalDescriptionItem({ title, body }: Props) {
+  const titleMd = `## ${title}`
```

Comment:
what is MD?

https://rollsafe.org/mdma/      👀

Reply:
`titleMarkdown` or `markdownTitle` is better :)

Reply:
Or maybe just `title` and `body` and add `Markdown` to the component name